recipes/Getting_Started_with_Granite_Code.ipynb

```diff
@@ -69,12 +72,57 @@
    "outputs": [],
    "source": [
     "import getpass, os\n",
+    "from ibm_granite_community.notebook_utils import is_colab\n",
     "\n",
-    "replicate_api_token = getpass.getpass()\n",
-    "\n",
-    "os.environ[\"REPLICATE_API_TOKEN\"] = replicate_api_token"
+    "if is_colab():\n",
```

Comment:
Can we also factor this into `utils` and lazily call if the user invokes a `find_langchain_model` with "replicate" as the model platform?